wp-cli-templates/block-register-php.mustache

```diff
@@ -7,10 +7,23 @@
  * @package Jetpack
  */
 
-jetpack_register_block(
-	'jetpack/{{ slug }}',
-	array( 'render_callback' => 'jetpack_{{ underscoredSlug }}_block_load_assets' )
-);
+namespace Jetpack\{{ title }}_Block;
```

Comment:
I wonder if this should be {{ slug }} too?

Reply:
Yes you're right, it needs to account for block names that may include spaces. c58458d302f701c147e774111bbaac51936bc516 should take care of that.